src/encoded/tests/test_indexing.py

```diff
@@ -88,7 +92,8 @@ def setup_and_teardown(app):
     # AFTER THE TEST
     session = app.registry[DBSESSION]
     connection = session.connection().connect()
```

Comment:
The change in how MetaData is used is one of the things nudged by the change in pytest version. It complained a lot about  us using a old mechanism.